bin/s2nc.c

```diff
@@ -60,7 +58,8 @@ void usage()
     fprintf(stderr, "  --ciphers [version_string]\n");
     fprintf(stderr, "    Set the cipher preference version string. Defaults to \"default\". See USAGE-GUIDE.md\n");
     fprintf(stderr, "  --enter-fips-mode\n");
-    fprintf(stderr, "    Enter libcrypto's FIPS mode. The linked version of OpenSSL must be built with the FIPS module.\n");
+    fprintf(stderr,
+            "    Enter libcrypto's FIPS mode. The linked version of OpenSSL must be built with the FIPS module.\n");
```

Comment:
You might want to disable formatting for the help message in both this and s2nd. It looks good with everything lined up properly, even if some lines are slightly too long.

If we're going to wrap this, let's do it by hand to keep the nice alignment.